examples/fuzz_trend_control_manager_20901.py

```diff
@@ -6,10 +6,14 @@
 # this was a really half assed fuzz. someone should take it further, see my notes in the requests file for more info.
 #
 
+from boofuzz import Session, Target, s_get
+
 # noinspection PyUnresolvedReferences
-from requests import trend  # noqa: F401
+# pytype: disable=import-error
+from request_definitions import trend  # noqa: F401
+
+# pytype: enable=import-error
 
-from boofuzz import *
```

Comment:
Does pytype flag `import *`s?

Reply:
Only when scanning single files as it seems to be missing the context then.
When running it on the whole project, star imports work fine.